### ISSUE TITLE
5/5 HF (watchlist) Killed Spy mission does not count for CS Espionage Mission

### ISSUE THREAD

**Issue by Stalker012345** [LoneGazebo/Community-Patch-DLL#7832]
_1. Mod version (i.e Date - 4/14):_
5/5 HF

_2. Mod list (if using Vox Populi only, leave blank):_

_3. Error description:_ 
This happened quick, so I'm not 100% sure it was a true issue, but I'm making people aware of it since it can happen rarely. I got a CS mission to do two spy missions on Askia. One completed by the spy was killed....and it did not count as a successful completion for the quest.

_4. Steps to reproduce (optional):_
![image](https://user-images.githubusercontent.com/20593976/117501030-21d1cf80-af43-11eb-8ea3-e4aa156d0977.png)

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. For instructions, go to the repository's main page, under "To enable logging for bug reports"):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves; you can change autosave frequency in the game's Options menu):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):

**Comment by Stalker012345** [comment]
This one is slippery, I still feel like there is an issue but its not all the time. I wonder if spies in capitals count and elsewhere doesn't or something, I'm not sure.

**Comment by LoneGazebo** [comment]
if you can get me a good save from when you think it happens, that would help. Quick glance at code shows no issue.

**Comment by Stalker012345** [comment]
Ok I've managed to get some secondary evidence. So in teh first screenshot, I need 6 spy missions to trigger the quest. I currently have 3 spies in spain, all in security 1 cities. 1 mission should complete next turn.

![image](https://user-images.githubusercontent.com/20593976/118585564-caabd600-b75e-11eb-90e4-32729e44be43.png)

![image](https://user-images.githubusercontent.com/20593976/118585614-e1eac380-b75e-11eb-8506-2793d24d598c.png)


The next turn, the spy finishes but is killed. Note that the quest counter has not gone down.

![image](https://user-images.githubusercontent.com/20593976/118585711-12caf880-b75f-11eb-932b-ec6ad7c2be07.png)

Fast forward 12 turns later, and even with none stop spies going to security 1 cities in spain....not a single change in the quest.

![image](https://user-images.githubusercontent.com/20593976/118585852-5291e000-b75f-11eb-9c73-8b55a707ad66.png)


Here's the other thing. I'm about 70% positive that quest on Turn 310 said 5 left, until I reloaded it. I want to say that the quest counter actually reset on my game reload, but I just can't say that with 100% confidence.

Regardless, there is definately something fishy here. I have attached save games and mod packet.
